lib/server/http/request_validation.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+/**
+* Checks if a duplicate value is passed through.
```

Comment:
I would change it to something like checks if duplicate keys are passed in the object. Add more descriptive text for what happens with Arrays and Arrays of objects

Reply:
Maybe add a couple of examples